### ISSUE TITLE
Will you kindly provide you training parameters? 

### ISSUE THREAD

**Issue by suikammd** [kedartatwawadi/NN_compression#8]
Sorry， I‘m new to train lstm. Will you kindly provide you training parameters? Since I can't get results as you provided. Or maybe it's because you didn't provide arithmetic coding block?

**Comment by kedartatwawadi** [comment]
Hello, the loss function should give the compression performance (bits/symbol) which would be achieved if arithmetic coding is used. 

See the updated README and https://web.stanford.edu/~kedart/files/deepzip.pdf for more info.
The default parameters in src/models/char_rnn.py worked well. If you want to explore hyperparameters, consider using src/scripts/run.py by appropriately modifying for your needs. (some examples are in the same folder)

**Comment by suikammd** [comment]
@kedartatwawadi So you finally use src/scripts/run.py to train DNA datasets and the model in src/models/char_rnn.py is your GRU-feature models?
Will kindly add a clear statement and tell us which one is your best performing network?
Thanks a lot!
